<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="EPINPUNW0043" ignored="4" name="Surefire test" tests="1" failures="1" timestamp="31 Jan 2019 08:48:15 GMT" time="3.185" errors="0">
  <testcase name="testWithInvalidUrl" time="3.139" classname="com.project.api.testscenarios.httpstatusvalidationtests.EmailValidation_GetCall">
    <failure type="java.lang.AssertionError" message="1 expectation failed.
Expected status code &amp;lt;404&amp;gt; doesn&amp;apos;t match actual status code &amp;lt;200&amp;gt;.
">
      <![CDATA[java.lang.AssertionError: 1 expectation failed.
Expected status code <404> doesn't match actual status code <200>.

at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrap.callConstructor(ConstructorSite.java:84)
at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallConstructor(CallSiteArray.java:60)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:235)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:247)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:451)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$1.call(Unknown Source)
at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:48)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:113)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)
at io.restassured.internal.ResponseSpecificationImpl.validateResponseIfRequired(ResponseSpecificationImpl.groovy:613)
at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:210)
at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:59)
at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:52)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:154)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:166)
at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:120)
at io.restassured.specification.ResponseSpecification$statusCode$0.callCurrent(Unknown Source)
at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:52)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:154)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:166)
at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:128)
at io.restassured.internal.ValidatableResponseOptionsImpl.statusCode(ValidatableResponseOptionsImpl.java:117)
at com.project.app.api_test_v1.utilities.RestCalls.assertResponseCode(RestCalls.java:291)
at com.project.api.testscenarios.httpstatusvalidationtests.EmailValidation_GetCall.testWithInvalidUrl(EmailValidation_GetCall.java:78)
... Removed 39 stack frames]]>
    </failure>
  </testcase> <!-- testWithInvalidUrl -->
  <testcase name="testWithDifferentCallMethod" time="0.0" classname="com.project.api.testscenarios.httpstatusvalidationtests.EmailValidation_GetCall">
    <ignored/>
  </testcase> <!-- testWithDifferentCallMethod -->
  <testcase name="successGetCallWithOtherValidEmail" time="0.0" classname="com.project.api.testscenarios.functionalvalidationtests.EmailValidation_GetCall">
    <ignored/>
  </testcase> <!-- successGetCallWithOtherValidEmail -->
  <testcase name="successGetCall" time="0.0" classname="com.project.api.testscenarios.functionalvalidationtests.EmailValidation_GetCall">
    <ignored/>
  </testcase> <!-- successGetCall -->
  <testcase name="getCallWithInvalidEmail" time="0.0" classname="com.project.api.testscenarios.functionalvalidationtests.EmailValidation_GetCall">
    <ignored/>
  </testcase> <!-- getCallWithInvalidEmail -->
</testsuite> <!-- Surefire test -->
